tests/by-util/test_hashsum.rs

```diff
@@ -29,6 +29,15 @@ macro_rules! test_digest {
                        get_hash!(ts.ucmd().arg(DIGEST_ARG).arg(BITS_ARG).pipe_in_fixture("input.txt").succeeds().no_stderr().stdout_str()));
         }
 
+        #[test]
+        fn test_nonames() {
+            let ts = TestScenario::new("hashsum");
+            // EXPECTED_FILE has no newline character but the tool output has a newline character
+            assert_eq!(ts.fixtures.read(EXPECTED_FILE),
+                       ts.ucmd().arg(DIGEST_ARG).arg(BITS_ARG).arg("--no-names").arg("input.txt").succeeds().no_stderr().stdout_str()
```

Comment:
I am pretty sure that rustfmt is going to complain :)

Reply:
I'm trying to make it complain but it's quite liberal about linebreaks, or?

Reply:
`rustfmt tests/by-util/test_hashsum.rs` passes without any changes (`rustfmt 1.4.38-stable (9d1b210 2022-02-23)`)

Reply:
I'll check the next CI results for details, not sure if I should move the `);` up to the end of the line and so on. Maybe because we are in a macro rustfmt can't operate as usual in my case?